docs/api-guide/serializers.md

```diff
@@ -419,6 +419,39 @@ You can provide arbitrary additional context by passing a `context` argument whe
 
 The context dictionary can be used within any serializer field logic, such as a custom `.to_representation()` method, by accessing the `self.context` attribute.
 
+From views, it's often useful to override `get_serializer_context` to add any additional data needed by your serializer to the context dict.  For example, here's a serializer that looks at the captured values from url, which come to the view as self.kwargs (keyword arguments to the constructor).
+
+    # serializers.py
+    class ThingSerializer(serializers.ModelSerializer):
+        ...
+        def create(self, validated_data):
+            value1 = validated_data['field1']
+            kwargs = self.context.get('kwargs')  # see views.py:get_serializer_context()
```

Comment:
It isn't really necessary to override `get_serializer_context()`. The `view` is already present in the context, which has its assigned `kwargs`. You could simplify this and remove the `get_serializer_context` override

```python 
    def create(self, validated_data):
        kwargs = self.context['view'].kwargs
        captured_value = kwargs['captured_arg'] if kwargs else None
        ...
```